### PR TITLE
Track errors in VoiceMessagePresenter

### PR DESCRIPTION
**Pull request by julioromano**
Story: https://github.com/vector-im/element-meta/issues/2085

### CODE REVIEW

File: features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/voicemessages/timeline/VoiceMessagePresenter.kt [hunk 1]
```diff
@@ -102,7 +105,19 @@ class VoiceMessagePresenter @AssistedInject constructor(
                     if (playerState.isPlaying) {
                         player.pause()
                     } else {
-                        scope.launch { play.runUpdatingState { player.play() } }
+                        scope.launch {
+                            play.runUpdatingState(
+                                errorTransform = {
+                                    it.apply {
```
Comment (bmarty) [inline features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/voicemessages/timeline/VoiceMessagePresenter.kt:111]: The `apply` is not necessary, I would simply write

```kotlin
errorTransform = {
    analyticsService.trackError(
        VoiceMessageException.PlayMessageError("Error while trying to play voice message", it)
    )
    it
},
```

to improve readability.

**Review by bmarty — APPROVED** [review]
LGTM modulo a minor nit comment.

File: features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/voicemessages/timeline/VoiceMessagePresenter.kt (hunk 1 above)
Comment (julioromano) [reply features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/voicemessages/timeline/VoiceMessagePresenter.kt:111]: I had it exactly like that before but it looked odd to my eyes so I switched to using `apply` 😆

**Comment by julioromano** [comment]
> LGTM modulo a minor nit comment.

Done

**Review by bmarty — APPROVED** [review]

**Pull request merged**